### PR TITLE
Add note about special case for Synthetic Test SLO

### PR DESCRIPTION
**Pull request by markazerdd**
### What does this PR do?

Adds a note about a special case for the SLO overall status calculation when a monitor-based SLO uses a single grouped Synthetic Test.

### Motivation

Customer support ticket regarding this calculation.

### Preview link

https://docs-staging.datadoghq.com/markazerdd/slo-synthetics-note/monitors/service_level_objectives/monitor/#overall-status-calculation

Check preview base path using the URL in details in `preview` status check.

### CODE REVIEW

File: content/en/monitors/service_level_objectives/monitor.md [hunk 1]
```diff
@@ -63,6 +63,17 @@ Consider the following example for 3 monitors:
 
 This can result in the overall status being lower than the average of the individual statuses.
 
+### Exceptions for Synthetic Tests
+In certain cases there is an exception to the status calculation for monitor-based SLOs that are comprised of one grouped Synthetic Test. Synthetic Tests have special alerting conditions that will change the behavior of when the test will enter the ALERT state. These conditions include:
```
Comment (MargotLepizzera) [inline content/en/monitors/service_level_objectives/monitor.md:67]:
```suggestion
In certain cases, there is an exception to the status calculation for monitor-based SLOs that are comprised of one grouped Synthetic test. Synthetic tests have optional special alerting conditions that will change the behavior of when the test will enter the ALERT state and consequently impact the overall uptime:
```

File: content/en/monitors/service_level_objectives/monitor.md [hunk 2]
```diff
@@ -63,6 +63,17 @@ Consider the following example for 3 monitors:
...
+
+- Wait until the groups are failing for a specified number of minutes (default: 0)
+- Wait until a specified number of the groups are failing (default: 1)
+- Retry a specified number of times before a location's test is considered a failure (default: 0)
+
+{{< img src="monitors/service_level_objectives/synthetic_alert_conditions.png" alt="synthetic test alert conditions"  >}}
+
+By changing these any of these conditions to something other than their defaults, the overall status for a monitor-based SLO using just that one Synthetic Test will appear to be better than the aggregated statuses of the Synthetic Test's individual groups.
```
Comment (MargotLepizzera) [inline content/en/monitors/service_level_objectives/monitor.md:75]:
```suggestion
By changing any of these conditions to something other than their defaults, the overall status for a monitor-based SLO using just that one Synthetic test could appear to be better than the aggregated statuses of the Synthetic test's individual groups. 
```

File: content/en/monitors/service_level_objectives/monitor.md [hunk 3]
```diff
@@ -63,6 +63,17 @@ Consider the following example for 3 monitors:
...
 
+### Exceptions for Synthetic Tests
+In certain cases there is an exception to the status calculation for monitor-based SLOs that are comprised of one grouped Synthetic Test. Synthetic Tests have special alerting conditions that will change the behavior of when the test will enter the ALERT state. These conditions include:
+
+- Wait until the groups are failing for a specified number of minutes (default: 0)
+- Wait until a specified number of the groups are failing (default: 1)
+- Retry a specified number of times before a location's test is considered a failure (default: 0)
+
+{{< img src="monitors/service_level_objectives/synthetic_alert_conditions.png" alt="synthetic test alert conditions"  >}}
+
```
Comment (MargotLepizzera) [inline content/en/monitors/service_level_objectives/monitor.md:74]: We could add a doc link to https://docs.datadoghq.com/synthetics/api_tests/?tab=httptest#alert-conditions

File: content/en/monitors/service_level_objectives/monitor.md [hunk 4]
```diff
@@ -63,6 +63,17 @@ Consider the following example for 3 monitors:
 
 This can result in the overall status being lower than the average of the individual statuses.
 
+### Exceptions for Synthetic Tests
+In certain cases, there is an exception to the status calculation for monitor-based SLOs that are comprised of one grouped Synthetic test. Synthetic tests have optional special alerting conditions that will change the behavior of when the test will enter the ALERT state and consequently impact the overall uptime:
```
Comment (cswatt) [inline content/en/monitors/service_level_objectives/monitor.md:67]:
```suggestion
In certain cases, there is an exception to the status calculation for monitor-based SLOs that are comprised of one grouped Synthetic test. Synthetic tests have optional special alerting conditions that change the behavior of when the test enters the ALERT state and consequently impact the overall uptime:
```

**Review by cswatt — APPROVED** [review]

**Pull request merged**